VMEncryption/main/DiskUtil.py

```diff
@@ -410,6 +510,33 @@ def _extract_luks_version_from_dump(self, luks_dump_out):
             if "version:" in line.lower():
                 return line.split()[-1]
 
+    def _extract_luksv2_token(self, luks_dump_out):
+        """
+        ...
+        Tokens:
+            1: Azure_Disk_Encryption_BackUp
+            5: Azure_Disk_Encryption
+        ...
+        """
```

Comment:
Could you please add a sample output of luks_dump_out? Its difficult to understand how the parsing works without a sample in the documentation.

Reply:
done